addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -10065,4 +10065,60 @@ QUnit.module("Views", (hooks) => {
         assert.containsN(target, ".o_kanban_record span", 3);
         assert.strictEqual(target.querySelector(".o_kanban_record").innerText, "ged");
     });
+
+    QUnit.test(
+        "keep focus inside control panel when 'focus-renderer' and no kanban card",
```

Comment:
nitpicking: "focus-renderer" is an implementation detail, I would rather say "when pressing arrowdown"